llvm-ops-metrics/ops-container/process_llvm_commits.py

```diff
@@ -42,12 +59,14 @@
 @dataclasses.dataclass
 class LLVMCommitInfo:
   commit_sha: str
-  commit_datetime: datetime.datetime
   commit_timestamp_seconds: int
+  files_modified: list[str]
+  commit_author: str = ""  # GitHub username of author is unknown until API call
   has_pull_request: bool = False
-  pr_number: int = 0
+  pull_request_number: int = 0
   is_reviewed: bool = False
   is_approved: bool = False
+  reviewers: set[str] = dataclasses.field(default_factory=set)
```

Comment:
Why the inconsistency here? You default initialize this, but don't default initialize `commit_sha`, commit_timestamp_seconds`, or `files_modified`. You also use a set here when both `files_modified` and `reviewers` could be represented as sets.